docker/candlepin-base/cp-test

```diff
@@ -105,16 +107,17 @@ OPTIONS:
                    suite and name
   -H               run rspec tests in "hosted" mode (implies -r and -d)
   -u               run unit test suite
```

Comment:
We should update this to indicate that specifying it multiple times triggers multiple runs